### DISCUSSION TITLE
Lost connection to MySQL server during query

### DISCUSSION THREAD

**Discussion by magongyong** [apache/doris#24195] [RFC]
版本2.0.1 release
执行一个时间长的sql，会报3次错误，Lost connection to MySQL server during query，每次报错时间间隔为5分钟
然后交互界面sql退出
但是实际上sql在后台会运行三次，如下截图
![image](https://github.com/apache/doris/assets/71626759/6b775fdd-0ebe-4eb3-9fe7-72bdf7d1cde3)

**Comment by magongyong** [comment]
因为这个sql是拉取tidb数据，所以可以在上游tidb可以查到的会话信息，如下截图
![image](https://github.com/apache/doris/assets/71626759/a3cb7c17-a689-4914-8f5a-e8f37eac87b1)

**Reply by yagagagaga** [reply]
What's client you use? Your client does not appear to be an official MySQL client.

**Reply by magongyong** [reply]
![image](https://github.com/apache/doris/assets/71626759/f4cfe713-12e3-45fb-ba4e-a2c1157c3c46)

**Reply by yagagagaga** [reply]
Do you have any disconnection parameters set on your server?

**Reply by magongyong** [reply]
I don't set any disconnection parameters .
this is be conf
![image](https://github.com/apache/doris/assets/71626759/27c3f4f9-3fa3-492b-871e-3d6a329d63c3)

**Comment by xyh-wiki** [comment]
请问此问题原因是什么, 如何解决的
